lib/core/include/rodsKeyWdDef.h

```diff
@@ -104,6 +104,7 @@
 #define REG_REPL_KW                                 "regRepl"  /* register replica */
 #define AGE_KW                                      "age"  /* age of the file for itrim */
 #define DRYRUN_KW                                   "dryrun"  /* do a dry run */
+#define NO_COMPUTE_KW                               "no_compute"  /* do not compute anything (similar to dryrun) */
```

Comment:
I notice this is the first(?) with an underscore... I think we want `noCompute` instead for consistency?

Reply:
It isn't the first, but I can change it if you like. I'm pretty sure there are other instances in that file.

Reply:
oooh, i see all the new ones have underscores.    resolving.   carry on.